From 6b01fd6b6b8d5a8a40e8dd210fd3b147a8941fa6 Mon Sep 17 00:00:00 2001
From: Mikolaj Izdebski <mizdebsk@redhat.com>
Date: Wed, 5 Dec 2012 12:39:00 +0100
Subject: [PATCH] Fix OSGi manifests (rhbz#883464)

According to OSGi specification Require-Bundle header should have
"bundle-version" attribute, not "version".
---
 batik-bridge-MANIFEST.MF     | 12 ++++++------
 batik-css-MANIFEST.MF        |  4 ++--
 batik-dom-MANIFEST.MF        |  6 +++---
 batik-dom-svg-MANIFEST.MF    |  6 +++---
 batik-ext-awt-MANIFEST.MF    |  4 ++--
 batik-extension-MANIFEST.MF  | 12 ++++++------
 batik-parser-MANIFEST.MF     |  6 +++---
 batik-svggen-MANIFEST.MF     |  4 ++--
 batik-swing-MANIFEST.MF      | 12 ++++++------
 batik-transcoder-MANIFEST.MF | 12 ++++++------
 batik-util-MANIFEST.MF       |  2 +-
 batik-xml-MANIFEST.MF        |  2 +-
 12 files changed, 41 insertions(+), 41 deletions(-)

diff --git a/batik-bridge-MANIFEST.MF b/batik-bridge-MANIFEST.MF
index 8d3e283..1428823 100644
--- a/batik-bridge-MANIFEST.MF
+++ b/batik-bridge-MANIFEST.MF
@@ -14,10 +14,10 @@ Export-Package: org.apache.batik.bridge;version="1.6.0",org.apache.bat
  n="1.6.0",org.apache.batik.gvt.svg12;version="1.6.0",org.apache.batik
  .gvt.text;version="1.6.0",org.apache.batik.script;version="1.6.0",org
  .apache.batik.script.rhino;version="1.6.0",META-INF.services
-Require-bundle: org.apache.batik.css;version="[1.6.0,1.7.0)",org.apach
- e.batik.dom;version="[1.6.0,1.7.0)",org.apache.batik.dom.svg;version=
- "[1.6.0,1.7.0)",org.apache.batik.ext.awt;version="[1.6.0,1.7.0)",org.
- apache.batik.parser;version="[1.6.0,1.7.0)",org.apache.batik.util;ver
- sion="[1.6.0,1.7.0)",org.apache.batik.util.gui;version="[1.6.0,1.7.0)
- ",org.apache.batik.xml;version="[1.6.0,1.7.0)",org.w3c.dom.svg;versio
+Require-bundle: org.apache.batik.css;bundle-version="[1.6.0,1.7.0)",org.apach
+ e.batik.dom;bundle-version="[1.6.0,1.7.0)",org.apache.batik.dom.svg;bundle-version=
+ "[1.6.0,1.7.0)",org.apache.batik.ext.awt;bundle-version="[1.6.0,1.7.0)",org.
+ apache.batik.parser;bundle-version="[1.6.0,1.7.0)",org.apache.batik.util;bundle-ver
+ sion="[1.6.0,1.7.0)",org.apache.batik.util.gui;bundle-version="[1.6.0,1.7.0)
+ ",org.apache.batik.xml;bundle-version="[1.6.0,1.7.0)",org.w3c.dom.svg;bundle-versio
  n="[1.1.0,1.3.0)"
diff --git a/batik-css-MANIFEST.MF b/batik-css-MANIFEST.MF
index 1e60bf6..29272d7 100644
--- a/batik-css-MANIFEST.MF
+++ b/batik-css-MANIFEST.MF
@@ -12,6 +12,6 @@ Export-Package: org.apache.batik.css.dom;version="1.6.0",org.apache.ba
  e.batik.css.engine.value.css2;version="1.6.0",org.apache.batik.css.en
  gine.value.svg;version="1.6.0",org.apache.batik.css.engine.value.svg1
  2;version="1.6.0",org.apache.batik.css.parser;version="1.6.0"
-Require-bundle: org.apache.batik.util;version="[1.6.0,1.7.0)",org.w3c.
- css.sac;version="[1.3.0,1.4.0)",org.w3c.dom.svg;version="[1.1.0,1.3.0
+Require-bundle: org.apache.batik.util;bundle-version="[1.6.0,1.7.0)",org.w3c.
+ css.sac;bundle-version="[1.3.0,1.4.0)",org.w3c.dom.svg;bundle-version="[1.1.0,1.3.0
  )"
diff --git a/batik-dom-MANIFEST.MF b/batik-dom-MANIFEST.MF
index f2929e6..0097206 100644
--- a/batik-dom-MANIFEST.MF
+++ b/batik-dom-MANIFEST.MF
@@ -11,6 +11,6 @@ Bundle-Vendor: %providerName
 Export-Package: org.apache.batik.dom;version="1.6.0",org.apache.batik.
  dom.events;version="1.6.0",org.apache.batik.dom.traversal;version="1.
  6.0",org.apache.batik.dom.util;version="1.6.0"
-Require-bundle: org.apache.batik.css;version="[1.6.0,1.7.0)",org.apach
- e.batik.util;version="[1.6.0,1.7.0)",org.apache.batik.xml;version="[1
- .6.0,1.7.0)",org.w3c.css.sac;version="[1.3.0,1.4.0)"
+Require-bundle: org.apache.batik.css;bundle-version="[1.6.0,1.7.0)",org.apach
+ e.batik.util;bundle-version="[1.6.0,1.7.0)",org.apache.batik.xml;bundle-version="[1
+ .6.0,1.7.0)",org.w3c.css.sac;bundle-version="[1.3.0,1.4.0)"
diff --git a/batik-dom-svg-MANIFEST.MF b/batik-dom-svg-MANIFEST.MF
index db03caa..77723b5 100644
--- a/batik-dom-svg-MANIFEST.MF
+++ b/batik-dom-svg-MANIFEST.MF
@@ -9,6 +9,6 @@ Bundle-Version: 1.6.0.v200805290154
 Bundle-Vendor: %providerName
 Export-Package: org.apache.batik.dom.svg;version="1.6.0",org.apache.ba
  tik.dom.svg12;version="1.6.0"
-Require-Bundle: org.apache.batik.css;version="[1.6.0,1.7.0)",org.apach
- e.batik.dom;version="[1.6.0,1.7.0)",org.apache.batik.parser;version="
- [1.6.0,1.7.0)",org.apache.batik.util;version="[1.6.0,1.7.0)",org.w3c.dom.svg;version="[1.1.0,1.3.0)"
+Require-Bundle: org.apache.batik.css;bundle-version="[1.6.0,1.7.0)",org.apach
+ e.batik.dom;bundle-version="[1.6.0,1.7.0)",org.apache.batik.parser;bundle-version="
+ [1.6.0,1.7.0)",org.apache.batik.util;bundle-version="[1.6.0,1.7.0)",org.w3c.dom.svg;bundle-version="[1.1.0,1.3.0)"
diff --git a/batik-ext-awt-MANIFEST.MF b/batik-ext-awt-MANIFEST.MF
index ef379d2..5f3a6cb 100644
--- a/batik-ext-awt-MANIFEST.MF
+++ b/batik-ext-awt-MANIFEST.MF
@@ -15,5 +15,5 @@ Export-Package: org.apache.batik.ext.awt;version="1.6.0",org.apache.ba
  .ext.awt.image.renderable;version="1.6.0",org.apache.batik.ext.awt.im
  age.rendered;version="1.6.0",org.apache.batik.ext.awt.image.spi;versi
  on="1.6.0",org.apache.batik.ext.swing;version="1.6.0"
-Require-Bundle: org.apache.batik.util;version="[1.6.0,1.7.0)",org.apac
- he.batik.util.gui;version="[1.6.0,1.7.0)"
+Require-Bundle: org.apache.batik.util;bundle-version="[1.6.0,1.7.0)",org.apac
+ he.batik.util.gui;bundle-version="[1.6.0,1.7.0)"
diff --git a/batik-extension-MANIFEST.MF b/batik-extension-MANIFEST.MF
index dc169dd..cc3967d 100644
--- a/batik-extension-MANIFEST.MF
+++ b/batik-extension-MANIFEST.MF
@@ -8,9 +8,9 @@ Bundle-Version: 1.6.0.v200805290154
 Bundle-Vendor: %providerName
 Export-Package: org.apache.batik.extension;version="1.6.0",org.apache.
  batik.extension.svg;version="1.6.0",META-INF.services
-Require-Bundle: org.apache.batik.bridge;version="[1.6.0,1.7.0)",org.ap
- ache.batik.css;version="[1.6.0,1.7.0)",org.apache.batik.dom;version="
- [1.6.0,1.7.0)",org.apache.batik.dom.svg;version="[1.6.0,1.7.0)",org.a
- pache.batik.ext.awt;version="[1.6.0,1.7.0)",org.apache.batik.parser;v
- ersion="[1.6.0,1.7.0)",org.apache.batik.util;version="[1.6.0,1.7.0)",
- org.w3c.dom.svg;version="[1.1.0,1.3.0)"
+Require-Bundle: org.apache.batik.bridge;bundle-version="[1.6.0,1.7.0)",org.ap
+ ache.batik.css;bundle-version="[1.6.0,1.7.0)",org.apache.batik.dom;bundle-version="
+ [1.6.0,1.7.0)",org.apache.batik.dom.svg;bundle-version="[1.6.0,1.7.0)",org.a
+ pache.batik.ext.awt;bundle-version="[1.6.0,1.7.0)",org.apache.batik.parser;bundle-v
+ ersion="[1.6.0,1.7.0)",org.apache.batik.util;bundle-version="[1.6.0,1.7.0)",
+ org.w3c.dom.svg;bundle-version="[1.1.0,1.3.0)"
diff --git a/batik-parser-MANIFEST.MF b/batik-parser-MANIFEST.MF
index a6a3586..d481070 100644
--- a/batik-parser-MANIFEST.MF
+++ b/batik-parser-MANIFEST.MF
@@ -7,6 +7,6 @@ Bundle-Localization: plugin
 Bundle-Version: 1.6.0.v200805290154
 Bundle-Vendor: %providerName
 Export-Package: org.apache.batik.parser;version="1.6.0"
-Require-Bundle: org.apache.batik.ext.awt;version="[1.6.0,1.7.0)",org.a
- pache.batik.util;version="[1.6.0,1.7.0)",org.apache.batik.xml;version
- ="[1.6.0,1.7.0)",org.w3c.dom.svg;version="[1.1.0,1.3.0)"
+Require-Bundle: org.apache.batik.ext.awt;bundle-version="[1.6.0,1.7.0)",org.a
+ pache.batik.util;bundle-version="[1.6.0,1.7.0)",org.apache.batik.xml;bundle-version
+ ="[1.6.0,1.7.0)",org.w3c.dom.svg;bundle-version="[1.1.0,1.3.0)"
diff --git a/batik-svggen-MANIFEST.MF b/batik-svggen-MANIFEST.MF
index 0426544..760f283 100644
--- a/batik-svggen-MANIFEST.MF
+++ b/batik-svggen-MANIFEST.MF
@@ -9,5 +9,5 @@ Bundle-Vendor: %providerName
 Export-Package: org.apache.batik.svggen;version="1.6.0",org.apache.bat
  ik.svggen.font;version="1.6.0",org.apache.batik.svggen.font.table;ver
  sion="1.6.0"
-Require-Bundle: org.apache.batik.ext.awt;version="[1.6.0,1.7.0)",org.a
- pache.batik.util;version="[1.6.0,1.7.0)"
+Require-Bundle: org.apache.batik.ext.awt;bundle-version="[1.6.0,1.7.0)",org.a
+ pache.batik.util;bundle-version="[1.6.0,1.7.0)"
diff --git a/batik-swing-MANIFEST.MF b/batik-swing-MANIFEST.MF
index 7965331..087b15f 100644
--- a/batik-swing-MANIFEST.MF
+++ b/batik-swing-MANIFEST.MF
@@ -9,9 +9,9 @@ Bundle-Vendor: %providerName
 Export-Package: org.apache.batik.swing;version="1.6.0",org.apache.bati
  k.swing.gvt;version="1.6.0",org.apache.batik.swing.svg;version="1.6.0
  "
-Require-Bundle: org.apache.batik.bridge;version="[1.6.0,1.7.0)",org.ap
- ache.batik.dom;version="[1.6.0,1.7.0)",org.apache.batik.dom.svg;versi
- on="[1.6.0,1.7.0)",org.apache.batik.ext.awt;version="[1.6.0,1.7.0)",o
- rg.apache.batik.util;version="[1.6.0,1.7.0)",org.apache.batik.util.gu
- i;version="[1.6.0,1.7.0)",org.apache.batik.xml;version="[1.6.0,1.7.0)
- ",org.w3c.dom.svg;version="[1.1.0,1.3.0)"
+Require-Bundle: org.apache.batik.bridge;bundle-version="[1.6.0,1.7.0)",org.ap
+ ache.batik.dom;bundle-version="[1.6.0,1.7.0)",org.apache.batik.dom.svg;bundle-versi
+ on="[1.6.0,1.7.0)",org.apache.batik.ext.awt;bundle-version="[1.6.0,1.7.0)",o
+ rg.apache.batik.util;bundle-version="[1.6.0,1.7.0)",org.apache.batik.util.gu
+ i;bundle-version="[1.6.0,1.7.0)",org.apache.batik.xml;bundle-version="[1.6.0,1.7.0)
+ ",org.w3c.dom.svg;bundle-version="[1.1.0,1.3.0)"
diff --git a/batik-transcoder-MANIFEST.MF b/batik-transcoder-MANIFEST.MF
index 4fbf384..63285c3 100644
--- a/batik-transcoder-MANIFEST.MF
+++ b/batik-transcoder-MANIFEST.MF
@@ -13,9 +13,9 @@ Export-Package: org.apache.batik.transcoder;version="1.6.0",org.apache
  he.batik.transcoder.svg2svg;version="1.6.0",org.apache.batik.transcod
  er.wmf;version="1.6.0",org.apache.batik.transcoder.wmf.tosvg;version=
  "1.6.0"
-Require-Bundle: org.apache.batik.bridge;version="[1.6.0,1.7.0)",org.ap
- ache.batik.dom;version="[1.6.0,1.7.0)",org.apache.batik.dom.svg;versi
- on="[1.6.0,1.7.0)",org.apache.batik.ext.awt;version="[1.6.0,1.7.0)",o
- rg.apache.batik.svggen;version="[1.6.0,1.7.0)",org.apache.batik.util;
- version="[1.6.0,1.7.0)",org.apache.batik.xml;version="[1.6.0,1.7.0)",
- org.w3c.dom.svg;version="[1.1.0,1.3.0)"
+Require-Bundle: org.apache.batik.bridge;bundle-version="[1.6.0,1.7.0)",org.ap
+ ache.batik.dom;bundle-version="[1.6.0,1.7.0)",org.apache.batik.dom.svg;bundle-versi
+ on="[1.6.0,1.7.0)",org.apache.batik.ext.awt;bundle-version="[1.6.0,1.7.0)",o
+ rg.apache.batik.svggen;bundle-version="[1.6.0,1.7.0)",org.apache.batik.util;
+ bundle-version="[1.6.0,1.7.0)",org.apache.batik.xml;bundle-version="[1.6.0,1.7.0)",
+ org.w3c.dom.svg;bundle-version="[1.1.0,1.3.0)"
diff --git a/batik-util-MANIFEST.MF b/batik-util-MANIFEST.MF
index 81c3ba3..e07afd3 100644
--- a/batik-util-MANIFEST.MF
+++ b/batik-util-MANIFEST.MF
@@ -10,4 +10,4 @@ Export-Package: org.apache.batik;version="1.6.0",org.apache.batik.i18n
  ;version="1.6.0",org.apache.batik.util;version="1.6.0",org.apache.bat
  ik.util.io;version="1.6.0",org.apache.batik.util.resources;version="1
  .6.0"
-Require-Bundle: org.apache.batik.util.gui;version="[1.6.0,1.7.0)"
+Require-Bundle: org.apache.batik.util.gui;bundle-version="[1.6.0,1.7.0)"
diff --git a/batik-xml-MANIFEST.MF b/batik-xml-MANIFEST.MF
index 94e913b..aba0d6f 100644
--- a/batik-xml-MANIFEST.MF
+++ b/batik-xml-MANIFEST.MF
@@ -7,4 +7,4 @@ Bundle-Localization: plugin
 Bundle-Version: 1.6.0.v200805290154
 Bundle-Vendor: %providerName
 Export-Package: org.apache.batik.xml;version="1.6.0"
-Require-Bundle: org.apache.batik.util;version="[1.6.0,1.7.0)"
+Require-Bundle: org.apache.batik.util;bundle-version="[1.6.0,1.7.0)"
--
1.8.0
